### PR TITLE
angsd: add 0.933, add misc programs to install

### PR DESCRIPTION
**Pull request by codeandkey**

### CODE REVIEW

File: var/spack/repos/builtin/packages/angsd/package.py [hunk 1]
```diff
@@ -16,18 +16,30 @@ class Angsd(MakefilePackage):
...
 
     depends_on('htslib')
     conflicts('^htslib@1.6:', when='@0.919')
 
+    depends_on('zlib')
+    depends_on('lzma')
+    depends_on('curl')
+
     def setup_run_environment(self, env):
         env.set('R_LIBS', self.prefix.R)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/angsd/package.py:35]: Are we missing a run-time dependency on R?

Comment (codeandkey) [reply var/spack/repos/builtin/packages/angsd/package.py:35]: `angsd` provides some R tools through this path but does not require R to run.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/angsd/package.py:35]: Hmm, I wonder if we should add a variant that optionally adds an R dependency. I don't use this package, so I'll leave it up to you to decide what's best.

Comment (codeandkey) [reply var/spack/repos/builtin/packages/angsd/package.py:35]: I went ahead and added an `r` variant enabling the runtime dependency.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**